network-store-model/src/main/java/com/powsybl/network/store/model/TwoWindingsTransformerAttributes.java

```diff
@@ -136,4 +137,10 @@ public class TwoWindingsTransformerAttributes extends AbstractAttributes impleme
 
     @Schema(description = "CGMES tap changer attributes list")
     private List<CgmesTapChangerAttributes> cgmesTapChangerAttributesList;
+
+    @Override
+    @JsonIgnore
+    public ResourceType getEquipmentType() {
```

Comment:
You don't need this, as you can get Resource by a backward link attributes.getResource().

Reply:
Yes, removed.